modules/yarn.lock

```diff
@@ -1747,6 +1747,7 @@
     babel-jest "^25.1.0"
     babel-loader "^8.1.0"
     babel-plugin-transform-react-remove-prop-types "0.4.24"
+    bourbon "4.2.3"
```

Comment:
Do we really need bourbon? I thought we already got rid of it... 🤔

Reply:
Yeah its used in a few places and the frontend-css-common module exposes it to the `buildCSS` task to consume in the sass build process